javascript/extractor/tests/html-text-extraction/output/trap/text-examples.html.trap

```diff
@@ -9,234 +9,234 @@ hasLocation(#10000,#10002)
 #20000=@"global_scope"
 scopes(#20000,0)
 #20001=*
-xmlElements(#20001,"html",#10000,0,#10000)
-#20002=@"loc,{#10000},1,1,25,7"
-locations_default(#20002,#10000,1,1,25,7)
+xmlChars(#20001,"
```

Comment:
I thought this would just be a refactor; why have test results changed?

Reply:
Tuples moved around because the extraction order changed.